ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json

```diff
@@ -84,8 +89,13 @@
     "AccessControl": [
       {
         "UserRoles": [
+          "APP-SOL-SPEC-PROFILE",
+          "APP-SOL-UNSPEC-PROFILE",
           "RES-SOL-ONE-SPEC-PROFILE",
-          "RES-SOL-ONE-UNSPEC-PROFILE"
+          "RES-SOL-ONE-UNSPEC-PROFILE",
+          "RES-SOL-TWO-SPEC-PROFILE",
+          "RES-SOL-TWO-UNSPEC-PROFILE",
+          "CITIZEN-DEFENDANT-PROFILE"
```

Comment:
Do we want all of these to be CRU?
I wonder if the applicant ones should be read-only.

Reply:
I don't know, to be honest. This is what @LeonardoPalmeiro advised me to do.

Reply:
I'm just not sure we want a respondent to have create/update access to an applicant field, and vice versa.

Reply:
Fixed.